build.sbt

```diff
@@ -183,8 +183,8 @@ lazy val algebird = Project(
   algebirdCore,
   algebirdUtil,
   algebirdBijection,
-  algebirdBenchmark,
-  algebirdSpark
+  algebirdBenchmark//,
+  //algebirdSpark
```

Comment:
let's not comment out algebirdSpark. It is a pain that `2.12.x` won't build, but CI knows about it. Can build with `sbt ++2.11.8 compile` on the command line.